polyfill/test/intl.mjs

```diff
@@ -1050,10 +1050,47 @@ describe('Intl', () => {
       it('should return an Array', () => assert(Array.isArray(Intl.DateTimeFormat.supportedLocalesOf())));
     });
 
-    const us = new Intl.DateTimeFormat('en-US', { timeZone: 'America/New_York' });
-    const at = new Intl.DateTimeFormat('de-AT', { timeZone: 'Europe/Vienna' });
+    // Verify that inputs to DateTimeFormat constructor are immune to mutation.
+    // Also verify that options properties are only read once.
+    const onlyOnce = (value) => {
+      const obj = {
+        calls: 0,
+        toString() {
+          if (++this.calls > 1) throw new RangeError('prop read twice');
+          return value;
+        },
+        toJSON() {
+          if (++this.calls > 1) throw new RangeError('prop read twice');
+          return value;
```

Comment:
Should this ever be called?

Reply:
Good catch. An early version of this PR used a JSON round-trip as a cheap way to do a deep copy of the input parameters, so this was previously useful. Now it's implemented differently so this `toJSON` test code will never run. I'll remove it, unless you think there's value in leaving it around in case some future implementation decides to use `JSON.stringify` on the inputs.

Reply:
I'm happy to keep it and add `valueOf` as well, or just drop it.

Reply:
I just dropped it for simplicity. We already have to revisit this per https://github.com/js-temporal/temporal-polyfill/pull/12#discussion_r670048689 so any improvement in multi-call/proxy-calll tests could happen then.

I think we're good to merge. It's pretty late for me so I'm going to bed soon. If I can't stay up until the Test262 run finishes, I'll merge in the morning if you don't do it first.  Then I can go on vacation! :-)